### PR TITLE
Nxo feature

### PR DESCRIPTION
**Pull request by KevinLay689**
Thank you for your contribution to Braintree. 

> Before submitting this PR, note that we cannot accept language translation PRs. We support the same languages that are supported by PayPal, and have a dedicated localization team to provide the translations. If there is an error in a specific translation, you may open an issue and we will escalate it to the localization team.

### Summary of changes

 - Creating a PR of the feature branch that contains: https://github.com/braintree/braintree_android/pull/560

 ### Checklist

 - [x] Added a changelog entry

- @kelay
- @tichow

### CODE REVIEW

**Review by sshropshire — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,31 @@
...
 
-## Unreleased
+## unreleased
 * Adds `invalidateClientToken` method to `BraintreeClient`, which invalidates the cached client
 token authorization provided by the `ClientTokenProvider`
 * Add `isExplicitCancelation` parameter to `UserCanceledException`
 * Trim tokenization key and client token before parsing
-
+* PayPalNativeCheckout
+  * Adding in [PayPalNativeCheckout] module to use the native checkout for PayPal
```
Comment (sshropshire) [inline CHANGELOG.md:9]:
```suggestion
  * Add new module to use Native Checkout for PayPal transactions
```

Comment (sshropshire) [reply CHANGELOG.md:9]: Nitpick. We rarely add new modules so we don't have a set practice for announcing new features via CHANGELOG.

**Review by jaxdesmarais — APPROVED** [review]
⛵

File: build.gradle [hunk 2]
```diff
@@ -33,7 +32,7 @@ buildscript {
             "kotlinCoroutinesCore"       : "org.jetbrains.kotlinx:kotlinx-coroutines-core:1.5.2",
 
             "browserSwitch"              : "com.braintreepayments.api:browser-switch:2.1.1",
-            "cardinal"                   : "org.jfrog.cardinalcommerce.gradle:cardinalmobilesdk:2.2.6-2",
+            "cardinal"                   : "org.jfrog.cardinalcommerce.gradle:cardinalmobilesdk:2.2.6-1",
```
Comment (sshropshire) [inline build.gradle:35]: Same here this version got bumped down.

File: Demo/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -119,4 +121,6 @@
 
     <!-- SamsungPay -->
     <string name="bt_start_samsung_pay">Start SamsungPay</string>
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
```
Comment (sshropshire) [inline Demo/src/main/res/values/strings.xml:125]:
```suggestion
```

**Review by sshropshire — APPROVED** [review]
🌊

**Pull request merged**